src/back-end/lib/permissions.ts

```diff
@@ -204,3 +204,9 @@ export async function editCWUProposal(connection: Connection, session: Session,
 export async function deleteCWUProposal(connection: Connection, session: Session, proposalId: string): Promise<boolean> {
   return session.user && await isCWUProposalAuthor(connection, session.user, proposalId) || false;
 }
+
+// Metrics.
+
+export function readAllCounters(session: Session): boolean {
+  return isAdmin(session);
+}
```

Comment:
I may not have explained this in the ticket, so my apologies if that's the case. Can we add a separate function called `readManyCounters`, and allow all admins & gov users to do so? This permissions check should happen at the beginning of the `readMany` CRUD actions. Thinking is that non-admin CWU gov opp owners will need to see an opportunity's total views.

Reply:
No problem, I will add this.  I'll take opportunityId from the name of the counter provided to determine whether a gov user is an owner of the opportunity corresponding to the counter.

Reply:
Actually, I realized that you probably meant all gov owners should be able to see all counters, not just the ones they own?

Reply:
Yeah, no need to do extra opp-based validation. Any gov user can view any counter.